.github/workflows/unit-tests.yaml

```diff
@@ -93,10 +93,17 @@ jobs:
       run: |
           ./run-checks --unit
 
+    - name: Create coverage results name
+      if: ${{ ! inputs.skip-coverage }}
+      run: |
+        converted=$(sed 's/\//-/g' <<<${{ inputs.gochannel }})
```

Comment:
btw you can also use `tr` for character manipulation and is usually simpler than sed, eg. `tr '/' '-'`